src/librustdoc/html/render.rs

```diff
@@ -2622,42 +2698,32 @@ fn item_trait(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
         }
 
         write!(w, "{}", impl_header)?;
+        for implementor in concrete {
+            render_implementor(cx, implementor, w, &implementor_dups)?;
+        }
+        write!(w, "</ul>")?;
```

Comment:
So in here you `<ul class='item-list' id='synthetic-implementors-list'>`.

Reply:
Actually, this closes `<ul class='item-list' id='implementors-list'>` from `impl_header`.